node/risk-app/server/services/__tests__/defineBackLink.spec.js

```diff
@@ -17,11 +18,11 @@ describe('defineBackLink', () => {
     expect(backLink).toEqual(postcodePage)
   })
 
-  test('Map page backlink takes user back to risk page', async () => {
-    const riskPage = '/risk'
+  test('Map page backlink takes user back to previous page', async () => {
```

Comment:
You will probably want more test cases as well, checking where the user is taken in each possible scenario (surface water/R&S/Reservoirs/no referred page)